vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -4069,6 +4445,126 @@ unit:DeciM
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "Decimeter" ;
 .
+unit:DeciM2
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1e-02 ;
+  qudt:conversionOffset 0e+00 ;
+  qudt:description "0,01-fold of the power of the SI base unit metre with the exponent 2" ;
+  qudt:hasQuantityKind quantitykind:Area ;
+  qudt:iec61360Code "0112/2///62720#UAA413" ;
+  qudt:uneceCommonCode "DMK" ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
+  rdfs:label "DeciM2" ;
+  skos:prefLabel "decimetre squared" ;
+.
+unit:DeciM3
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1e-03 ;
+  qudt:conversionOffset 0e+00 ;
+  qudt:description "0,001-fold of the power of the SI base unit metre with the exponent 3" ;
```

Comment:
Similar to the DeciM2 unit, I would say "One tenth of a meter, cubed"
Same logic would apply to all the DeciM... units.

Reply:
fixed